src/redux/reducer.js

```diff
@@ -14,12 +15,13 @@ const persistConfig = {
 const MainReducer = persistCombineReducers(persistConfig, {
     auth: AuthReducer,
     user: UserReducer,
+    school: SchoolReducer,
 });
 
 const RootReducer = (state, action) => {
     if (action.type === AuthActionTypes.LOGOUT) {
         storage.removeItem("persist:auth");
-        state.auth = state.user = undefined;
+        state.auth = state.user = state.school = undefined;
```

Comment:
A thought. What if you retrieve _persist and clear state and set it back? This wouldnt require changes over time when new modules are added

https://github.com/VPPR/frontend/blob/9d9a2463a6f2b72a7be0aa62f995f7a0467f2077/src/redux/reducer.js#L25

Reply:
that's probably a good idea. Might make a `misc` PR for small changes like this one....